eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -197,8 +196,10 @@ conwayGovCertTransition = do
           }
   where
     checkColdCredHasNotResigned coldCred csCommitteeCreds =
-      ((isNothing <$> Map.lookup coldCred csCommitteeCreds) /= Just True)
-        ?! ConwayCommitteeHasPreviouslyResigned coldCred
+      case Map.lookup coldCred csCommitteeCreds of
+        Just (CommitteeMemberResigned _) -> failBecause $ ConwayCommitteeHasPreviouslyResigned coldCred
```

Comment:
Couldn't decide whether to add the anchor in the predicate failure - if anyone can thinks it could be useful, I'm happy to add it

Reply:
Nah, I don't think it would useful, since neither presence nor contents of the anchor affect the predicate check